.devcontainer/devcontainer.json

```diff
@@ -4,10 +4,7 @@
 
   "build": {
     "dockerfile": "Dockerfile",
-    "context": "..",
-    "args": {
-      "VARIANT": "20.18.1-bookworm"
-    }
+    "context": ".."
```

Comment:
While we could just update the value of this VARIANT argument and leave the Dockerfile as-is, I think it will be less confusing for future updates if we just remove this and update the Dockerfile value.

Reply:
Sounds good. Fine with me.